docs/cbdb-op-deploy-guide.md

```diff
@@ -2,10 +2,707 @@
+
+    The modified example configuration file is as follows:
+
+    ```toml
+    #### Base number by which mirror segment port numbers 
+    #### are calculated.
+    MIRROR_PORT_BASE=17000
+    
+    #### File system location(s) where mirror segment data directories 
+    #### will be created. The number of mirror locations must equal the
+    #### number of primary locations as specified in the 
+    #### DATA_DIRECTORY parameter.
+    declare -a MIRROR_DATA_DIRECTORY=(data0/mirror)
+    ```
+
+#### Step 6: Initialize the database
+
+Initialize the database using the `gpinitsystem` command:
+
+```bash
+gpinitsystem -c gpinitsystem_config -p cbdb_etcd.conf -F fts_service.conf  -E etcd_service.conf -h seg_host -s standby
+```
+
+### Hybrid deployment mode
+
+In this mode, you can reuse the existing physical machines to deploy the FTS and ETCD services. No additional physical machines are required for FTS and ETCD. The system reliability is not as high as that of the standard distributed deployment. This deployment mode supports automatic fault recovery by switching between master/standby nodes.
+
+To learn the physical configuration required for this mode, see [configuration for production environment](./cbdb-op-software-hardware.md#production-environment).
+
+To deploy Cloudberry Database in this mode, take the following steps.
+
+#### Step 1: Plan the deployment
+
+It is recommended to plan the nodes to be deployed according to the following table:
+
+| Component       | Recommended number of physical machines | Description |
+| :-------------  | :---------------- | :---------- |
+| Master node     | 1                |             |
+| Standby node    | 1                | Standby nodes are used for hot backup of master nodes. |
+| Segment (computing node) |  3      | It is recommended to deploy the same number of mirror nodes as computing nodes for high availability. The number of nodes is determined according to your data requirements and related hardware (the number of disks mounted on the machine). |
+| FTS node        | 3                | The FTS cluster supports multi-node hybrid deployment, with 3 nodes as the default configuration to ensure high availability. You do not need to reserve a separate physical machine for it. |
+| ETCD metadata node | 3             | The ETCD cluster supports multi-node deployment, with high availability feature natively supported by the application. You do not need to reserve a separate physical machine for it. |
+
+:::info
+
+- The following deployment operations are all performed with the `gpadmin` user.
+- Because segments are computing nodes, deploying the ETCD and FTS services on computing nodes might cause performance issues in production environments. To avoid performance issues and improve system reliability, it is recommended to specify an additional physical machine `{host}` through the configuration file to deploy ETCD and FTS services, that is, master + standby + {host} deployment.
```

Comment:
在 [Google developer documentation style guide](https://developers.google.com/style/word-list#letter-b) 中，"since" 是有歧义的单词，也可以表示时间关系。
![image](https://github.com/cloudberrydb/cloudberrydb-site/assets/41534398/18134966-e191-456d-a7ca-26cbe1617db8)